components/table/demo/table-test.js

```diff
@@ -98,7 +99,7 @@ class TestTable extends RtlMixin(DemoPassthroughMixin(TableWrapper, 'd2l-table-w
 					</thead>
 					<tbody>
 						<tr class="d2l-table-header">
-							<th scope="col" sticky></th>
+							<th scope="col" sticky><d2l-button-icon icon="tier1:gear"></d2l-button-icon></th>
```

Comment:
We generally want to create examples that look like "real world" usages. e.g., for this one, we would want it within one of the header cells with content rather than the checkbox cell.
Also, this doesn't actually have bottom spacing as it is now, only top.